utilities/cli/tests/test_cli.py

```
@@ -286,35 +286,18 @@ def test_lint_fix_command(self):
 
     @unittest.skipIf(not is_cookiecutter_available(), "cookiecutter not installed")
     @patch("utilities.cli.holohub.HoloHubCLI._install_template_deps")
-    @patch("cookiecutter.main.cookiecutter")
-    @patch("utilities.cli.holohub.HoloHubCLI.handle_create")
+    @patch("cookiecutter.main.cookiecutter", return_value="/tmp/dummy_project")
+    @patch("utilities.cli.holohub.HoloHubCLI.validate_generated_metadata")
     @patch("utilities.cli.holohub.HoloHubCLI._add_to_cmakelists")
     def test_language_passed_to_cookiecutter(
         self,
         mock_add_to_cmakelists,
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_add_to_cmakelists`

(ARG002)